staging/src/k8s.io/apiserver/pkg/server/httplog/log.go

```diff
@@ -158,7 +146,7 @@ func (rl *respLogger) Addf(format string, data ...interface{}) {
 // Log is intended to be called once at the end of your request handler, via defer
 func (rl *respLogger) Log() {
 	latency := time.Since(rl.startTime)
-	if glog.V(2) {
```

Comment:
not sure about this one. This gives the htaccess log like lines, right? Many people want them I assume. Now those will get more noise with `--v=3`.